tests/python-api-client/testapi.py

```diff
@@ -0,0 +1,20 @@
+from lpaapi import *
```

Comment:
As a next step, would be nice to add some GETs to check that the API has added what you expect (out of scope for this issue but could raise separately).

Reply:
Sounds a good idea :)